src/js/api/config.js

```diff
@@ -89,13 +87,8 @@ const Config = function(options, persisted) {
     config.aspectratio = _evaluateAspectRatio(config.aspectratio, config.width);
 
     if (_.isObject(config.skin)) {
-        config.skinColors = normalizeSkin(config.skin);
         config.skinUrl = config.skin.url; // get skin name if it exists
-        config.skin = _.isString(config.skin.name) ? config.skin.name : Defaults.skin; // get skin name if it exists
```

Comment:
`config.skin` is an object so why store `skinName` in the config and model? You always normalize the model skin object so this is unnecessary.